src/lib.rs

```diff
@@ -162,23 +165,26 @@ mod hints;
 mod miniver;
 mod notification;
 mod timeout;
+pub(crate) mod urgency;
```

Comment:
please leave this at the bottom with the other `pub mod`s

Reply:
Added by rust-ananlyser. Might need a rustfmt.toml